node_modules/col-core/lib/server.js

```diff
@@ -17,14 +17,13 @@ var _ = require('lodash');
 var bodyParser = require('body-parser');
 var config = require('config');
 var cookieParser = require('cookie-parser');
+var cors = require('cors');
```

Comment:
Added it to `package.json` for now. I'll remove as part of the PR that introduce Apache and the production build. Just trying to keep things a bit staged for now.